server/policies/collection.test.ts

```diff
@@ -49,7 +32,7 @@ describe("admin", () => {
     expect(abilities.archive).toBeTruthy();
   });
 
-  it("should allow updating documents in view only collection", async () => {
+  it("should have correct permissions in view only collection", async () => {
```

Comment:
I removed the duplicate test block. The only distinction was that this one explicitly set the collection’s permissions to `read`, while the default behavior of createCollection is `read_write`, which implicitly includes `read`.